bolt/lib/Core/DebugNames.cpp

```diff
@@ -523,8 +523,9 @@ void DWARF5AcceleratorTable::writeEntries() {
               &EntriesBuffer.get()->data()[PatchOffset]);
           *Ptr = Iter->second;
         } else {
-          BC.errs() << "Could not find entry with offset " << *ParentOffset
-                    << "\n";
+          BC.errs() << "BOLT-WARNING: [internal-dwarf-warning]: Could not find "
```

Comment:
"Could" -> "could"

Under what conditions can the user get this warning? Bad input?

Reply:
I can't think of any. Unless there a bug.